imager/imagerapp/tests.py

```diff
@@ -61,3 +57,96 @@ def test_unicode_and_str(self):
         bill_unicode = unicode(bill.profile)
         self.assertEqual(isinstance(bill_str, str), True)
         self.assertEqual(isinstance(bill_unicode, unicode), True)
+
+
+class ImagerFollowTestCase(TestCase):
+    def setUp(self):
+        self.bill = User(username='bill')
+        self.sally = User(username='sally')
+        self.tracy = User(username='tracy')
+        self.bill.save()
+        self.sally.save()
+        self.tracy.save()
+
+    def test_followers_empty(self):
+        """ test to makes sure followers works on an empty set"""
+        sally = self.sally.profile
+        bill = self.bill.profile
+        self.assertEqual(bill.followers.count(), 0)
+        self.assertEqual(sally.followers.count(), 0)
+        self.assertEqual(bool(bill.followers.all()), False)
+        self.assertEqual(bool(sally.followers.all()), False)
```

Comment:
Why only test those two.  Why not all three?  Why not use a loop so you don't have to repeat so much:

```python
for user in self.users.values():
    self.assertEqual(user.followers.count(), 0)
    ...
```
Also I'm not sure the two separate assertions you have here are really any different from each-other.